sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -614,6 +619,7 @@ async def create_topic(self, topic_name: str, **kwargs) -> TopicProperties:
             availability_status=None,
             enable_express=kwargs.pop("enable_express", None),
             user_metadata=kwargs.pop("user_metadata", None),
+            max_message_size_in_kilobytes=kwargs.pop("max_message_size_in_kilobytes", None)
```

Comment:
I can't seem to spot where this is being used other than as a property like `self.max_message_size_in_kilobytes`. Is it meant to be used as a constraint somewhere? so like (` if size(message)< self.max_message_size_in_kilobytes:`)

Reply:
ah, just checked the issue - it says it's all service side validation. In that case, this is something the user sets so that the service can validate with this value?

Reply:
yes, and to be more precise on `user sets` -- user can create/update queue/topic of the property max_message_size_in_kilobytes being specified.

Reply:
makes sense!